libs/community/langchain_community/document_loaders/pebblo.py

```diff
@@ -46,6 +46,7 @@ def __init__(
         api_key: Optional[str] = None,
         load_semantic: bool = False,
         classifier_url: Optional[str] = None,
+        classifier_location: str = "local",
```

Comment:
Use `*` prior to new named arguments

Reply:
Added as suggested. 

@eyurtsev , Can you please help me understanding how is this going to help please. Is this a new norm that I am unaware of, or is it langchain coding style, please.